rotkehlchen/utils/snapshots.py

```diff
@@ -0,0 +1,98 @@
+from csv import DictReader
+from pathlib import Path
+from typing import Any, Dict, List, Tuple
+
+from rotkehlchen.assets.asset import Asset
+from rotkehlchen.constants.assets import A_USD
+from rotkehlchen.constants.misc import ONE
+from rotkehlchen.db.dbhandler import DBHandler
+from rotkehlchen.errors.price import NoPriceForGivenTimestamp
+from rotkehlchen.errors.serialization import DeserializationError
+from rotkehlchen.history.price import PriceHistorian
+from rotkehlchen.serialization.deserialize import deserialize_timestamp
+from rotkehlchen.types import Price, Timestamp
+from rotkehlchen.user_messages import MessagesAggregator
+
+
+def validate_import_data(
+        balances_data: List[Dict[str, str]],
+        location_data: List[Dict[str, str]],
+) -> Tuple[bool, str]:
+    """Validates the snapshot data about to be imported."""
```

Comment:
Can you also write in the docstring what this function can return? And what exceptions it may raise? I guess no exceptions but still checking.